src/Features/CSharp/Portable/CodeCleanup/CSharpCodeCleanupService.cs

```diff
@@ -178,6 +188,10 @@ internal class CSharpCodeCleanupService : ICodeCleanupService
 
                 // Mark this option as being completed.
                 progressTracker.ItemCompleted();
+
+                var result = await Formatter.FormatAsync(document).ConfigureAwait(false);
```

Comment:
is there reason this is added?

Reply:
Sorry, things got really messed up with all of the merge conflicts :(  I'll fix it.